tldw_Server_API/app/core/TTS/adapters/kitten_tts_adapter.py

```
@@ -0,0 +1,236 @@
+                    model_name,
+                    cache_dir=self.cache_dir,
+                    auto_download=self.auto_download,
+                    revision=self.model_revision,
+                )
+                runtime = await asyncio.to_thread(KittenRuntime, assets)
+            except Exception as exc:
+                raise TTSProviderInitializationError(
+                    "Failed to initialize KittenTTS runtime",
+                    provider=self.PROVIDER_KEY,
+                    details={"model": model_name, "error": str(exc)},
+                ) from exc
```

Ruff (0.15.6)

[warning] Avoid specifying long messages outside the exception class

(TRY003)